.github/actions/setup-build/action.yml

```diff
@@ -0,0 +1,85 @@
+name: Setup build environment
+description: Setup the build environment
+
+inputs:
+  node:
+    description: Setup node and install root node modules
+    default: false
+    required: false
+  subprojects:
+    description: Setup node and install root and subproject node modules (overrides `node` input)
+    default: false
+    required: false
+  ios-build:
+    description: Setup xcode and install dependencies
+    default: false
+    required: false
+  restore-pods:
+    description: Restores the Pod cache
+    default: false
+    required: false
+  java:
+    description: Setup Java and Gradle cache
+    default: false
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Set up Java
+      if: ${{ inputs.java == 'true' }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'zulu'
+        java-version: '17'
+
+    - name: Set up Gradle cache
+      if: ${{ inputs.java == 'true' }}
+      uses: gradle/gradle-build-action@v2
+      with:
+        cache-read-only: ${{ github.ref != 'refs/heads/development' }}
+
+    - name: Setup node and npm registry
+      if: ${{ inputs.node == 'true' || inputs.subprojects == 'true' }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20'
+        registry-url: 'https://registry.npmjs.org/'
+        cache: 'yarn'
+
+    - name: Install node_modules
+      if: ${{ inputs.node == 'true' || inputs.subprojects == 'true' }}
+      shell: bash
+      run: yarn install --frozen-lockfile
+
+    - name: Install node_modules (example/)
+      if: ${{ inputs.subprojects == 'true' || inputs.subprojects == 'true' }}
```

Comment:
Is this a typo? `inputs.subprojects == 'true'` is there 2x!

Reply:
Embarrassing. Seems to be a copy past error on my side. should be just once there.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/408/commits/443a4be8423bfb1c314ceef4760ff22aa6502365